projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.html

```diff
@@ -26,22 +24,43 @@ <h4>
       Here we'd like to introduce configurable facet components,
       either by using specific configuration or generic sproutlets
   -->
-  <cx-facet
-    *ngFor="let facet of facets"
-    #facetRef
-    [facet]="facet"
-    [class.expanded]="isExpanded(facet) | async"
-    [class.collapsed]="isCollapsed(facet) | async"
-    role="group"
-    attr.aria-label="{{
-      'productFacetNavigation.ariaLabelItemsAvailable'
-        | cxTranslate
-          : {
-              name: facet.name,
-              count: facet?.values?.length,
-            }
-    }}"
-  ></cx-facet>
+  <ng-container *cxFeature="'a11yTabComponent'">
+    <cx-tab [tabs$]="tabs$" [config]="tabConfig" *ngIf="tabs$ | async"></cx-tab>
```

Comment:
The `tabs$` is being subscribed by inside the `<cx-tab>` component, I think having this second subscription is redundant(`tabs$ | async`) . Do you think it will work if you just pass `*ngIf="tabs$"`

Reply:
Yup, seems to work without *ngIf at all